vuepress/docs/next/docs/curate/bundle-comparison.md

```diff
@@ -15,21 +15,22 @@ Some of the major improvements with docker-based bundles are:
-|Plugin Ingress| Generated by Entando or set in the plugin descriptor.yaml | Generated by Entando or customized in the [bundle descriptor](bundle-details.md#entando-bundle-conventions), entando.json 
-|Identifiers| Designated Bundle ID and Plugin ID | Bundles, MFEs, & Plugins have unique global IDs|
+| Bundle specifications |Defined with descriptor files | Built with [ent bundle CLI](../getting-started/ent-bundle.md) tool
+| Versioning |Bundle version set by Git tags |Bundle version set by Docker tags. For MFEs, the version defaults to 1.0.0 unless set in the [entando.json](bundle-details.md)
+|Plugin ingress| Generated by Entando or set in the plugin descriptor.yaml | Generated by Entando or customized in the [bundle descriptor](bundle-details.md#entando-bundle-conventions), entando.json
+|Identifiers| Designated Bundle ID and Plugin ID | Bundles, MFEs, & plugins have unique global IDs|
+|Bundle ID| Not included in component names | Suffix of bundle, plugin, widget, page and fragment names
 |Initialize from Entando Hub| As git clone  |[Direct initialization from Hub](../getting-started/ent-bundle.md#initialization) with `–from-hub` flag|
-|Bundle Creation | [`ent prj` commands](../getting-started/ent-bundle.md#git-based-bundle-commands)| New modular [`ent bundle` commands](../getting-started/ent-bundle.md)
-|Deploy & Install| `ent ecr deploy` and `ent ecr install` | No change 
-|Export Bundle|`ent bundler`| Not Available 
-|JHipster Integrated Blueprint Project| `ent jhipster blueprint` creates [MFE & MS template project](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md)| [Convert Blueprint project to v5 bundle](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) <!-- TODO-change link to new tutorial -->
-|API Claims| Manual configuration in MFE bundle | [Manage APIs with `ent bundle` commands](../getting-started/ent-api.md). MFEs can make API claims to MSs within the same bundle or namespace
-|Local Services |Blueprint includes a few services under src/main/docker. | Default services now include Keycloak, PostgreSQL, and MySQL. Additional services can be added with the [`ent bundle svc` commands](../getting-started/ent-svc.md).
-|MFE Custom UI| Add with manual scripts| Custom UI is auto-generated in FTL format
-|MFE Configuration| Manually defined in the FTL | Utilize context parameters, system parameters and the `ext` variable to pass information to widgets and bundles, or expose it in the APIs
-|| | Auto-generated FTLs with parameters defined in `entando.json` |
-|Bundle Thumbnail| Detailed inside the YAML bundle descriptor| Set with a JPG or PNG file named `thumbnail` in the bundle root folder, e.g. thumbnail.png
+|Bundle creation | [`ent prj` commands](../getting-started/ent-bundle.md#git-based-bundle-commands)| New modular [`ent bundle` commands](../getting-started/ent-bundle.md)
```

Comment:
Everything in the first column was capitalized because they are row headers to give it some emphasis.

Reply:
the column is labeled "Attribute" and capitalizing the row entries implies that each of those terms is an attribute, which is incorrect